src/bitcoind.cpp

```diff
@@ -159,6 +159,14 @@ bool AppInit(int argc, char* argv[])
             return false;
 #endif // HAVE_DECL_DAEMON
         }
+        // Warn about relative -datadir path.
```

Comment:
Any reason this is not in AppInitMain instead? I am asking, because I don't see the warning when using the gui.

Reply:
> Any reason this is not in AppInitMain instead? I am asking, because I don't see the warning when using the gui.

No reason, and this is a good thought. Moved in 46c40bfe2dbfc4ea05258c9b90273ed28cdc3278